applications/sandbox/systems/testsystem.hpp

```diff
@@ -757,18 +792,19 @@ class TestSystem final : public System<TestSystem>
             auto pos = ent2.get_component_handle<position>().read();
             pos.y = 6;
             ent2.get_component_handle<position>().write(pos);
+
         }
-#if defined(LEGION_DEBUG)
-        for (int i = 0; i < 2000; i++)
-#else
-        for (int i = 0; i < 20000; i++)
-#endif
-        {
-            auto ent = createEntity();
-            ent.add_components<rendering::mesh_renderable>(mesh_filter(billboardH.get_mesh()), rendering::mesh_renderer(textureBillboardH));
-            ent.add_component<sah>({});
-            ent.add_components<transform>(position(math::linearRand(math::vec3(40, -21, -10), math::vec3(60, -1, 10))), rotation(), scale(0.1f));
-        }
+//#if defined(LEGION_DEBUG)
+//        for (int i = 0; i < 2000; i++)
+//#else
+//        for (int i = 0; i < 20000; i++)
+//#endif
+//        {
+//            auto ent = createEntity();
+//            ent.add_components<rendering::mesh_renderable>(mesh_filter(billboardH.get_mesh()), rendering::mesh_renderer(textureBillboardH));
+//            ent.add_component<sah>({});
+//            ent.add_components<transform>(position(math::linearRand(math::vec3(40, -21, -10), math::vec3(60, -1, 10))), rotation(), scale(0.1f));
+//        }
```

Comment:
we just discussed a better solution with changing the `createEntity` function

Reply:
done